### PR TITLE
Ticket 4 get article by ID ready for review

### PR DESCRIPTION
**Pull request by adamjamo**
Ticket 4 get article by ID ready for review

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -29,3 +29,62 @@ describe("NC News Topics", () => {
...
+          topic: "mitch",
+          author: "icellusedkars",
+          body: "some gifs",
+          created_at: "2020-11-03T09:12:00.000Z",
+          votes: 0,
+        });
+      });
+  });
+  test("responds with article by ID", () => {
+    const article_id = 4;
```
Comment (HeatherEver) [inline __tests__/app.test.js]: this test is just doing the same as above - not really needed

File: __tests__/app.test.js [hunk 2]
```diff
@@ -29,3 +29,62 @@ describe("NC News Topics", () => {
...
+          title: "Student SUES Mitch!",
+          topic: "mitch",
+          author: "rogersop",
+          body: "We all love Mitch and his wonderful, unique typing style. However, the volume of his typing has ALLEGEDLY burst another students eardrums, and they are now suing for damages",
+          created_at: "2020-05-06T01:14:00.000Z",
+          votes: 0,
+        });
+      });
+  });
+});
```
Comment (HeatherEver) [inline __tests__/app.test.js:93]: tests for errors need to be added, think about what happens if 
- the article id is 999999
- the article id is the wrong data type

File: db/controllers/topics_controllers.js [hunk 3]
```diff
@@ -1,7 +1,25 @@
...
+} = require("../models/topics_models.js");
 
 exports.getTopics = (req, res, next) => {
   fetchTopics().then((topics) => {
     res.status(200).send({ topics });
   });
 };
+
+exports.getArticles = (req, res) => {
+  console.log("inside controller");
```
Comment (HeatherEver) [inline db/controllers/topics_controllers.js]: delete any old console logs

File: db/models/topics_models.js [hunk 4]
```diff
@@ -5,3 +5,39 @@ exports.fetchTopics = () => {
...
   });
 };
+
+exports.fetchArticles = () => {
+  return db.query(`SELECT * FROM articles;`).then((results) => {
+    return results.rows;
+  });
+};
+
+// exports.fetchArticlesById = (article_id) => {
```
Comment (HeatherEver) [inline db/models/topics_models.js]: commented out code should either be removed or used

File: db/models/topics_models.js [hunk 5]
```diff
@@ -5,3 +5,39 @@ exports.fetchTopics = () => {
...
+//           status: 404,
+//           errorMessage: "article ${article_id} does not exist",
+//         });
+//       }
+//       return rows[0];
+//     });
+// };
+
+exports.fetchArticlesById = (article_id) => {
+  console.log(article_id, "<<<< article_id");
```
Comment (HeatherEver) [inline db/models/topics_models.js]: another console log to delete

**Review by HeatherEver — COMMENTED** [review]
Good work just a few changes to make throughout your code and then think about your error handling for those instances i've suggested and how you can test and handle them in your code 👍

**Review by Angaca — APPROVED** [review]
This looks good, ready to go

**Pull request merged**